cygnus-common/src/main/java/com/telefonica/iot/cygnus/management/MetricsHandlers.java

```diff
@@ -0,0 +1,213 @@
+/**
+ * Copyright 2017 Telefonica Investigación y Desarrollo, S.A.U
+ *
+ * This file is part of fiware-cygnus (FIWARE project).
+ *
+ * fiware-cygnus is free software: you can redistribute it and/or modify it under the terms of the GNU Affero
+ * General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your
+ * option) any later version.
+ * fiware-cygnus is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the
+ * implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License
+ * for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License along with fiware-cygnus. If not, see
+ * http://www.gnu.org/licenses/.
+ *
+ * For those usages not covered by the GNU Affero General Public License please contact with iot_support at tid dot es
+ */
+package com.telefonica.iot.cygnus.management;
+
+import com.google.common.collect.ImmutableMap;
+import com.telefonica.iot.cygnus.handlers.CygnusHandler;
+import com.telefonica.iot.cygnus.metrics.CygnusMetrics;
+import com.telefonica.iot.cygnus.log.CygnusLogger;
+import com.telefonica.iot.cygnus.sinks.CygnusSink;
+import java.io.IOException;
+import java.lang.reflect.Field;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.apache.flume.Sink;
+import org.apache.flume.SinkProcessor;
+import org.apache.flume.SinkRunner;
+import org.apache.flume.Source;
+import org.apache.flume.SourceRunner;
+import org.apache.flume.source.http.HTTPSourceHandler;
+
+/**
+ *
+ * @author frb
+ */
+public final class MetricsHandlers {
+    
+    private static final CygnusLogger LOGGER = new CygnusLogger(MetricsHandlers.class);
+    
+    /**
+     * Constructor. Utility classes should not have a public or default constructor.
+     */
+    private MetricsHandlers() {
+    } // MetricsHandlers
+    
+    /**
+     * Handles GET /v1/admin/mergeMetrics and /admin/mergeMetrics.
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1383/commits/1ca87236b3c2b0b44789f1a3d2cf6801be79fb8d